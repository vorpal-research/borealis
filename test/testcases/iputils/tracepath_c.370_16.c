//
// Created by ice-phoenix on 9/18/15.
//

#include "defines.h"

#include <stdlib.h>

int main(int argc, char** argv) {
    int base_port = argc;
    char* p;

    ASSUME(argv);
    ASSUME(argv[0]);

    if ( ! base_port) {
        p = strchr(argv[0], '/');
        if (p) {
            *p = 0;
            base_port = atoi(p+1);
        } else
            base_port = 44444;
    }

    return base_port;
}
