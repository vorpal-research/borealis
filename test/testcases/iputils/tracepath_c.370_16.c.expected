[
    {
        "location": {
            "loc": {
                "col": 25,
                "line": 20
            },
            "filename": "test/testcases/iputils/tracepath_c.370_16.c"
        },
        "type": "BUF-01"
    }
]